script/upgrades/MU01/governance/Config.sol

```diff
@@ -354,4 +359,68 @@ library Config {
       config.minimumReports = 2;
     }
   }
+
+  function cEURUSDCConfig(Contracts.Cache storage contracts, uint8 phase) internal returns (PoolConfiguration memory config) {
+    require(phase >= 1 && phase <= 3, "phase must be 1, 2, or 3");
+    config = PoolConfiguration({
+      asset0: contracts.celoRegistry("StableTokenEUR"),
+      asset1: contracts.dependency("BridgedUSDC"),
+      isConstantSum: true,
+      spread: FixidityLib.newFixedFraction(25, 10000), // 0.0025
+      minimumReports: 5,
+      referenceRateResetFrequency: 5 minutes,
+      stablePoolResetSize: 1_800_000 * 1e18, // 12 million
+      isMedianDeltaBreakerEnabled: true,
+      medianDeltaBreakerThreshold: FixidityLib.newFixedFraction(2, 100), // 0.02
+      medianDeltaBreakerCooldown: 15 minutes,
+      smoothingFactor: FixidityLib.newFixedFraction(5, 10000), // 0.0005
+      isValueDeltaBreakerEnabled: true,
```

Comment:
@ninocomputer @bayological `cEURUSDC` `valueDeltaBreaker` should be referencing `USDCUSD`. I don't know the name of  the parameter.

Reply:
rateFeedDependencies